Content.Shared/Radio/EntitySystems/EncryptionKeySystem.cs

```diff
@@ -100,30 +122,21 @@ private void OnInteractUsing(EntityUid uid, EncryptionKeyHolderComponent compone
         if (!TryComp<ToolComponent>(args.Used, out var tool) || !tool.Qualities.Contains(component.KeysExtractionMethod))
             return;
 
-        args.Handled = true;
```

Comment:
if something generates pop ups it needs to mark the interaciton as handled, otherwise you get pop-up spam

Reply:
Fxied

Reply:
Fixed